src/aof.c

```diff
@@ -996,7 +996,7 @@ int startAppendOnly(void) {
 
         if (rewriteAppendOnlyFileBackground() == C_ERR) {
             server.aof_state = AOF_OFF;
-            serverLog(LL_WARNING,"Redis needs to enable the AOF but can't trigger a background AOF rewrite operation. Check the above logs for more info about the error.");
+            serverLog(LL_WARNING,"Server needs to enable the AOF but can't trigger a background AOF rewrite operation. Check the above logs for more info about the error.");
```

Comment:
This is part of #207 so I will tag @0del because he is assigned to it. @0del are you still working on the remaining logging?

In #207 we discussed this style:

    serverLog(LL_WARNING,"%s needs to enable the AOF but can't trigger a background AOF rewrite operation. Check the above logs for more info about the error.",
              SERVER_TITLE);

... because then it would be easy patch the SERVER_TITLE macro to be "Redis" to get the old logging. But then we discussed it a bit more and decided that most of the logging isn't a problem about redis compatibility, so I think we don't need this complexity.

I'm OK with "Server", but probably better grammar is "The server" in this case.

Reply:
Thank you for the review, Changed the string "Server" to "The server" and updated the diff.